src/shared/logs-show.c

```diff
@@ -511,13 +544,18 @@ static int output_short(
                 if (r < 0)
                         return r;
         }
+
         if (IN_SET(r, -EBADMSG, -EADDRNOTAVAIL)) {
                 log_debug_errno(r, "Skipping message we can't read: %m");
                 return 0;
         }
         if (r < 0)
                 return log_error_errno(r, "Failed to get journal fields: %m");
 
+        // for update_display_timestamps_and_bootid()
+        update_display_timestamps_and_bootid(j, (dual_timestamp *)display_ts, (sd_id128_t *)boot_id,
```

Comment:
No no, please do not update constant arguments.

Reply:
Yes, I understand this is problematic. I will add a comment below with an explanation.